### PR TITLE
Adding One New Test in New Client Record Form

### LINKED ISSUE #787: Adding One New Test in New Client Record Form
Currently in Indonesia, we have another test called GeNose C19, the result will be positive or negative. Could we add that to the type of test question below?

Screenshot_2021-02-16-08-32-49-359_io ona diagnostics rdt configurable id

@mwkhan1983 @vincent-karuri @syedowaisali90

### PR DESCRIPTION
**Pull request by syedowaisali90**
close #787

### CODE REVIEW

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json [hunk 1]
```diff
@@ -739,6 +739,35 @@
             }
           }
         }
+      },
+      {
+        "key": "genose_c19",
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json:744]: Shouldn't this be under type of test?

Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json:744]: done.

**Review by vincent-karuri — APPROVED** [review]
